docs/app/Examples/addons/TextArea/Types/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+
+const TextAreaTypesExamples = () => (
+  <ExampleSection title='Types'>
```

Comment:
We mirror Semantic-UI core doc headings and examples.  Then, all extra examples we're placing under a "Usage" heading, for now.  Let's update `Types` here to `Usage`, since Semantic-UI has no CSS definition page that shows "Types" for a textarea.